### PR TITLE
Add `MinValueTest` and remove `main` from `MinValue`

### PR DESCRIPTION
**Pull request by lukasb1b**
Adding Test to MinValue

- [x] I have read [CONTRIBUTING.md](https://github.com/TheAlgorithms/Java/blob/master/CONTRIBUTING.md).
- [x] This pull request is all my own work -- I have not plagiarized it.
- [x] All filenames are in PascalCase.
- [x] All functions and variable names follow Java naming conventions.
- [x] All new algorithms have a URL in their comments that points to Wikipedia or other similar explanations.

### CODE REVIEW

**Comment by vil02** [comment]
@lukasb1b where are the tests?

**Comment by lukasb1b** [comment]
@vil02 sry I forgot to mearch a branch

File: src/test/java/com/thealgorithms/maths/MinValueTest.java [hunk 1]
```diff
@@ -0,0 +1,13 @@
+package com.thealgorithms.maths;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+
+import org.junit.jupiter.api.Test;
+
+public class MinValueTest {
+    @Test
+    public void flipBitTest() {
```
Comment (vil02) [inline src/test/java/com/thealgorithms/maths/MinValueTest.java:9]:
```suggestion
    public void minTest() {
```

File: src/test/java/com/thealgorithms/maths/MinValueTest.java [hunk 2]
```diff
@@ -0,0 +1,13 @@
...
+        assertEquals(-1, MinValue.min(-1, 3));
+        assertEquals(2, MinValue.min(3, 2));
+    }
```
Comment (vil02) [inline src/test/java/com/thealgorithms/maths/MinValueTest.java:12]:
```suggestion
        assertEquals(5, MinValue.min(5, 5));
    }
```

File: src/main/java/com/thealgorithms/maths/MinValue.java [hunk 3]
```diff
@@ -1,24 +1,6 @@
 package com.thealgorithms.maths;
 
-import java.util.Random;
-
 public class MinValue {
```
Comment (vil02) [inline src/main/java/com/thealgorithms/maths/MinValue.java:3]:
```suggestion
public final class MinValue {
    private MinValue() {
    }
```

**Review by vil02 — CHANGES_REQUESTED** [review]
I would use the opportunity and update the doc-string.

**Comment by lukasb1b** [comment]
Sure upadte it. I think you should be able to change the file.

**Review by vil02 — APPROVED** [review]

**Pull request merged**